app/jobs/scheduled/pending_users_reminder.rb

```diff
@@ -3,13 +3,13 @@
 module Jobs
 
   class PendingUsersReminder < ::Jobs::Scheduled
-    every 1.hour
+    every 20.minutes
```

Comment:
Is there a topic discussing this change? Unless we run a job at a much higher frequency like 5 minutes, I don't think this change is currently worth making. This is because most of the site setting values wouldn't make any difference since the job is only ran thrice an hour.

Reply:
Yes this is a requested feature and I asked Roman to work on it. I agree we might need to go more frequent here (5-10 mins) though.